palm/plugins/dbt/dbt_palm_utils.py

```diff
@@ -51,6 +52,7 @@ def dbt_env_vars(branch: str) -> Dict:
         'PDP_DEV_SCHEMA': _generate_schema_from_branch(branch),
         'PDP_ENV': 'DEVELOPMENT',  # Deprecated - this will be removed!
         'PALM_DBT_ENV': 'DEVELOPMENT',
+        'DBT_'
```

Comment:
remove, add the env vars only in the commands you want, and only when you need them!